selfdrive/test/process_replay/test_processes.py

```diff
@@ -72,9 +72,9 @@ def run_test_process(data):
   return (segment, cfg.proc_name, res)
 
 
-def get_logreader(segment):
+def get_logreader(segment, cache):
```

Comment:
This is not needed! Use `os.environ["FILEREADER_CACHE"] = "1"` cache all files downloaded using URLFile which is used by logreader.

I think it would be useful to add these instructions somewhere

Reply:
FILEREADER_CACHE="1" scons -j8 && selfdrive/test/process_replay/test_processes.py --whitelist-cars SUBARU

should this work? 
it's not caching for me

Reply:
Nevermind, i see this doesn't persist so it shouldn't interfere with anything?